specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/examples/SelectiveRestore-example.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "keyName": "hsm-mail-key",
+    "restoreBlobDetails": {
+      "sasTokenParameters": {
+        "storageResourceUri": "https://myaccount.blob.core.windows.net/sascontainer/sasContainer",
+        "token": "se=2018-02-01T00%3A00Z&spr=https&sv=2017-04-17&sr=b&sig=XXFNfuMCH112BxhemJ7PWn0xGfImMXT6LfbXWvtRUk%3D"
+      },
+      "folderToRestore": "1490790332"
```

Comment:
Should this be renamed to `folder` perhaps? It otherwise sounds like the whole folder will be restored.